### DISCUSSION TITLE
Be able to disable a combination of a product

### DISCUSSION THREAD

**Discussion by alfonsolr09** [PrestaShop/PrestaShop#36588] [Ideas]
### The problem

You cannot disable a combination of a product, you can only delete it

### The solution

Be able to disable a combination of a product

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
It's a very big change for very little gain, in my opinion.

**Reply by alfonsolr09** [reply]
It is something that can be done right now in a simple product. Why not be able to do it in one with combinations?

**Reply by alfonsolr09** [reply]
It is something that can be done right now in a simple product. Why not be able to do it in one with combinations?
